component-processor/src/main/java/org/infinispan/component/processor/Model.java

```diff
@@ -190,13 +190,13 @@ public MOperation(String name, ManagedOperation operation, String returnType, Li
 
    public static class MParameter {
       final String name;
-      final String description;
       final String type;
+      final String description;
 
-      public MParameter(String name, String description, String type) {
+      public MParameter(String name, String type, String description) {
```

Comment:
I preferred changing the order here (to also match the order in its sister JmxOperationParameter) rather than fixing the call site.